reana_server/config.py

```diff
@@ -135,10 +134,32 @@ def _(x):
 APP_DEFAULT_SECURE_HEADERS["content_security_policy"] = {}
 APP_HEALTH_BLUEPRINT_ENABLED = False
 
-# Rate limiting configuration
+# Rate limiting configuration using invenio-app
 # ===========================
-RATELIMIT_AUTHENTICATED_USER = "20 per second"
-RATELIMIT_GUEST_USER = "20 per second"
+
+
+def _is_valid_rate_limit(rate_limit: str) -> bool:
+    return bool(
+        re.match(r"[0-9]+(\sper\s|\/)(second|minute|hour|day|month|year)", rate_limit,)
```

Comment:
I don't think we need the trailing comma here.

Reply:
Done